powerauth-push-model/src/main/java/io/getlime/push/model/validator/RemoveApnsRequestValidator.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.push.model.validator;
+
+import io.getlime.push.model.request.RemoveApnsRequest;
+
+/**
+ * Validator class for remove APNs request.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+public class RemoveApnsRequestValidator {
```

Comment:
🤔 Couldn't we use bean validation?

Reply:
Yes, I can switch to bean validation for all validators.